project/Build.scala

```diff
@@ -1341,12 +1341,14 @@ object Build {
       BuildInfoPlugin.buildInfoDefaultSettings
 
   def presentationCompilerSettings(implicit mode: Mode) = {
-    val mtagsVersion = "1.3.1"
+    val mtagsVersion = "1.3.1+63-1a8f4659-SNAPSHOT"
```

Comment:
Should wait for a stable metals release.

Reply:
We can update to 1.3.2 now :tada: